### PR TITLE
Upgrade modeljson ints to int64 if their proto uses int64

### PR DESCRIPTION
**Pull request by dmathieu**
This is part of https://github.com/elastic/apm-data/issues/118.

Some proto definitions are using int64 when their modeljson has been using int.
I'm doing it this way since the flow is modelpb->modeljson.

### CODE REVIEW

File: input/elasticapm/internal/modeldecoder/nullable/nullable.go [hunk 1]
```diff
@@ -193,6 +202,31 @@ func (v *Int) Reset() {
 	v.isSet = false
 }
 
+// Int64 stores an int64 value and the
+// information if the value has been set
+type Int64 struct {
```
Comment (dmathieu) [inline input/elasticapm/internal/modeldecoder/nullable/nullable.go:207]: There is a `generator/nullable`, but it seems unrelated to this. And running `make generate` doesn't regenerate this file. So while I don't know what that generator does, I guess it doesn't generate this.

**Comment by dmathieu** [comment]
I'm opening the PR this way after discussing it quickly with @kruskall due to the way the transformation happens.
However, for all these attributes, it seems unlikely the count needs to be an int64. So I'd be fine using int32 in the protobuf definitions too.

File: input/elasticapm/internal/modeldecoder/nullable/nullable.go (hunk 1 above)
Comment (dmathieu) [reply input/elasticapm/internal/modeldecoder/nullable/nullable.go:207]: Nevermind, the generator seems to only kick in when `nullable.Int64` is used.

**Review by axw — APPROVED** [review]

**Comment by axw** [comment]
> However, for all these attributes, it seems unlikely the count needs to be an int64. So I'd be fine using int32 in the protobuf definitions too.

Maybe should, not sure - but I think keeping them aligned makes sense either way, so we could do that as a separate change.
